pkg/Microsoft.Private.PackageBaseline/baseline.packages.props

```diff
@@ -323,7 +323,7 @@
       <Version>4.0.0</Version>
     </BaseLinePackage>
     <BaseLinePackage Include="System.Security.Cryptography.X509Certificates">
-      <Version>4.1.0</Version>
+      <Version>4.1.1</Version>
```

Comment:
Don't update this one.  That will cause all other packages to update their dependencies on your package.